.semaphore/semaphore.yml

```diff
@@ -9,44 +9,37 @@ agent:
     type: e1-standard-2
     os_image: ubuntu1804
 
+global_job_config:
+  prologue:
+    commands:
+      - checkout
+      - cache restore go-pkg-cache
+      - cache restore go-mod-cache
+      - 'cache restore bin-${SEMAPHORE_GIT_SHA}'
+
 blocks:
   - name: 'Build'
     dependencies: []
     task:
       jobs:
         - name: Build
           commands:
-            - checkout
-            - cache restore go-pkg-cache
-            - cache restore go-mod-cache
+            - make build
             - 'cache store bin-${SEMAPHORE_GIT_SHA} bin'
             - cache store go-pkg-cache .go-pkg-cache
```

Comment:
are the single-quotes around some of the `cache` commands required for macro/variable expansion?

Reply:
Yes it's for the ${SEMAPHORE_GIT_SHA} expansion